src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/NotifichePGPagoPATest.java

```diff
@@ -67,7 +67,7 @@ public void siVisualizzaCorrettamenteLaPaginaNotifichePersonaGiuridica(String ra
         piattaformaNotifichePGPAPage.waitLoadPiattaformaNotificaPage(ragioneSociale);
 
         String variabileAmbiente = System.getProperty("environment");
-        String urlChiamata = WebTool.getApiBaseUrl() + "notifications/received?";
+        String urlChiamata = "https://webapi." + variabileAmbiente + ".notifichedigitali.it/delivery/notifications/received?";
```

Comment:
The url has changed, check if you can use the one set in ApiBaseUrl, maybe the response is different...